src/courses/advanced/12.md

```diff
@@ -8,16 +8,21 @@ headerDepth: 3
 ## Next Steps
 
 ### Take the Class Survey
-Take our brief [survey](https://forms.office.com/g/W2xtcV2frW) to give feedback to fuel class improvement.
 
-### Save your work on GitHub
-If you want to save your work in your remote repository in GitHub, you need to use Git commands. You can reference a [Git cheat sheet](https://education.github.com/git-cheat-sheet-education.pdf) or checkout [this Git tutorial](https://learngitbranching.js.org/).
+Please take our brief [survey](https://forms.office.com/g/W2xtcV2frW) to provide feedback and help us improve the class.
 
-### Reference other class content
-This class is one of a set of security automation content offered by the MITRE SAF(c) team. If you found this content interesting and you want to learn more, we encourage you to go back to the User Class or Beginner Security Automation Developer Class (shown in the table of contents on the left).
+### Save Your Work on GitHub
+
+To save your work in your remote repository on GitHub, use Git commands. You can reference a [Git cheat sheet](https://education.github.com/git-cheat-sheet-education.pdf) or check out [this Git tutorial](https://learngitbranching.js.org/).
```

Comment:
should probably still do add instructions like for now, you can `cd to root dir`, `git add .`, and `git commit --signoff -m "MITRE ROCKS!"`